tools/riscv/README_RISCV.md

```diff
@@ -0,0 +1,45 @@
+# Building RISC-V Target with Cmake #
```

Comment:
Can you rename `tools/riscv/README_RISCV.md` to `arch/riscv/README.md` similar to s390?

Reply:
Sure, it's done.